src/components/NetPromotingScore/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+const NetPromotingScore = ({ start, end }) => {
+  const startScore = start || 1;
+  const endScore = end || 10;
+  const [nps, setNps] = useState(Math.ceil((startScore + endScore) / 2));
+  const scoreElements = Array.from(
+    { length: endScore - startScore + 1 },
+    (v, k) => k + startScore
+  );
+
+  return (
+    <div className="np-score">
+      <ul className="np-score__number-container">
+        { scoreElements.map((score, index) => {
+          const ratingValue = index + 1;
+
+          return (
+            <li
+              role="presentation"
+              key={ index }
+              className={
+                ratingValue === scoreElements.length
+                  ? 'np-score__number-label np-score__number-label--no-border'
+                  : 'np-score__number-label'
+              }
+              onClick={ () => setNps(score) }
+              onKeyDown={ () => setNps(score) }
+            >
+              <p
+                className={
+                  ratingValue <= nps
+                    ? 'np-score__number np-score__number--selected'
+                    : 'np-score__number'
+                }
+              >
+                { score }
+              </p>
+            </li>
+          );
+        }) }
+        <div className="np-score__identifier">
```

Comment:
@rafayet-monon  Should we move this out of the `ul` so when the mouse hovered on them the score opacity won't change
![npshover](https://user-images.githubusercontent.com/1772999/102443106-8c4ba700-4058-11eb-8553-09d2a30fb768.gif)